common/src/main/scala/db/DatabaseConfig.scala

```diff
@@ -32,10 +32,12 @@ object DatabaseConfig {
   private def transactorAndDataSource[F[_] : Async](config: JdbcConfig)(implicit cs: ContextShift[F]): (Transactor[F], HikariDataSource) = {
     val connectEC = ExecutionContext.fromExecutor(Executors.newFixedThreadPool(config.fixedThreadPool))
     val transactEC = ExecutionContext.fromExecutor(Executors.newCachedThreadPool)
-    val dataSource = new HikariDataSource()
-    dataSource.setJdbcUrl(config.url)
-    dataSource.setUsername(config.user)
-    dataSource.setPassword(config.password)
+    val ikariConfig = new HikariConfig()
```

Comment:
missing `h` at the beginning of `hikari`?

Reply:
Or could just be called `config`?